src/pkg/packager/dev.go

```diff
@@ -112,12 +113,14 @@ func (p *Packager) DevDeploy(ctx context.Context) error {
 
 // Lint ensures a package is valid & follows suggested conventions
 func (p *Packager) Lint(ctx context.Context) error {
-	config.CommonOptions.Confirm = true
-
 	if err := os.Chdir(p.cfg.CreateOpts.BaseDir); err != nil {
 		return fmt.Errorf("unable to access directory %q: %w", p.cfg.CreateOpts.BaseDir, err)
 	}
 
+	if err := utils.ReadYaml(layout.ZarfYAML, &p.cfg.Pkg); err != nil {
+		return err
+	}
```

Comment:
What do you think about changing `Validate` to accept a `ZarfPackage` struct as an argument, like we do with `lintComponents`?

This way, we can read the zarf.yaml file once and pass the pkg object to `Validate`:

```golang
Validate(ctx, p.cfg.Pkg,  p.cfg.CreateOpts)
```
Seems odd and inefficient to read the zarf.yaml twice here

It would avoid reading the file twice and make `Validate` easier to test

Reply:
Yeah the only thing is that validate will have to assume the package it gets has not already been composed, but I think a comment on the function explaining that will be sufficient

Reply:
Updated